src/main/java/org/fxmisc/flowless/VirtualFlowContent.java

```diff
@@ -177,6 +186,44 @@ void scrollY(double deltaY) {
         orientation.scrollVertically(this, deltaY);
     }
 
+    void scrollXToPixel(double pixel) {
+        orientation.scrollHorizontallyToPixel(this, pixel);
+    }
+
+    void scrollYToPixel(double pixel) {
+        orientation.scrollVerticallyToPixel(this, pixel);
+    }
+
+    Val<Double> totalWidthEstimateProperty() {
+        return Val.wrap(orientation.widthEstimateProperty(this));
```

Comment:
I initially wrote it like you suggested, but the IDE detected a type compatibility error. I guess it returns a ObservableValue<Double>. So that's why I wrapped it. 
Should I change the return type to Val<Double> ?